small-rna.wdl

```diff
@@ -0,0 +1,102 @@
+version 1.0
+
+# MIT License
+#
+# Copyright (c) 2018 Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import "structs.wdl" as structs
+import "sample.wdl" as SampleWorkflow
+
+workflow SmallRna {
+    input {
+        File sampleConfigFile
+        String outputDir = "."
+        Array[File] bowtieIndexFiles
+        String? platform = "illumina"
+        Array[File] gtfFiles
```

Comment:
This should be: `Array[File]+`, since that's the type of the Sample workflow input.

Do we really need to support multiple gtf files, though?

Reply:
Yes. Leon wants two gtf files to be tested on his data.